src/time-picker/index.tsx

```diff
@@ -11,6 +13,15 @@ export type Time = {
   minutes: number;
 };
 
+/**
+ * Time display type
+ */
+export type TimeDisplay = {
+  hours: number;
+  minutes: number;
+  amPm?: 'AM' | 'PM'
```

Comment:
Let's call this property `meridiem` - the actual name of what is either AM/PM.

And since we are going to be using `'AM'` and `'PM'` quite a lot in the code, creating an enum would be a great idea

```ts
enum Meridiem {
  AM = 'AM',
  PM = 'PM',
}
```

We can use these enums in other places following this.